### PR TITLE
Improved errors when loading ledger balance

### LINKED ISSUE #4920: Account balance: error, can't retrieve data
**Did you search for similar issues before submitting this one?**

**Describe the issue you encountered:**

Open payment window to see my account balance and I see this: error, can't retrieve data

**Expected behavior:**

See my account balance
- Platform (Win7, 8, 10? macOS? Linux distro?): Ubuntu 64
- Brave Version: Brave: 0.12.5
- Screenshot if needed:
  captura de tela de 2016-10-18 20-19-29
- Any related issues:

### PR DESCRIPTION
**Pull request by ayumi**
Humanizes errors when loading ledger balance by mapping error codes to localized messages.

Fix #4920

Auditors: @luixxiul

Test Plan:
1. Disable internet for Brave (can disable system wide, or app specific eg w/ Little Snitch).
2. Open Brave and view Preferences > Payments.
3. Account balance should show error and details.

### CODE REVIEW

File: app/extensions/brave/locales/bn-IN/preferences.properties [hunk 1]
```diff
@@ -26,7 +26,7 @@ paymentsSidebarText2=All transaction IP addresses are anonymized with technology
 paymentsSidebarText3=Brave Bitcoin Wallets are provided through a partnership with:
 paymentsSidebarText4=Your contributions in the form of credit cards and bank cards are handled by:
 accountBalance=account balance
-accountBalanceConnectionError=error, can't retrieve data
```
Comment (bbondy) [inline app/extensions/brave/locales/bn-IN/preferences.properties]: For l10n updates pls only update en-US the other files will be pulled down from transifex.

Comment (ayumi) [inline app/extensions/brave/locales/bn-IN/preferences.properties]: Thanks! updated the PR.

**Comment by bbondy** [comment]
lgtm but I didn't test, @luixxiul would you mind? Thanks.

**Comment by bbondy** [comment]
It's on master now, and will be in the 0.12.9 preview build.

**Pull request merged**